packages/instantsearch.js/src/components/SearchBox/SearchBox.tsx

```diff
@@ -183,6 +183,7 @@ class SearchBox extends Component<
             onInput={this.onInput}
             onBlur={this.onBlur}
             onFocus={this.onFocus}
+            aria-label="Search for results"
```

Comment:
Searchbox is also used to search for filters, probably this should be a prop used by SearchBox widget + RefinementList widget